package.json

```diff
@@ -30,10 +30,22 @@
     "remark-preset-lint-markdown-style-guide": "^3.0.0",
     "remark-preset-lint-recommended": "^4.0.0",
     "stylelint": "^13.6.1",
-    "stylelint-config-standard": "^20.0.0"
+    "stylelint-config-standard": "^20.0.0",
+    "@istanbuljs/nyc-config-typescript": "^1.0.1",
+    "nyc": "^15.1.0",
+    "source-map-support": "^0.5.19",
+    "typescript": "^4.0.2",
+    "ts-node": "^9.0.0",
+    "coveralls": "^3.1.0"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "coverage": "npm run instrument && npm run cp && npm run rm",
+    "instrument": "nyc instrument cvat-ui cvat-ui_cov && nyc instrument cvat-canvas cvat-canvas_cov && nyc instrument cvat-data cvat-data_cov && nyc instrument cvat-core cvat-core_cov",
```

Comment:
The config is based on ``@istanbuljs/nyc-config-typescript``, but cvat-core and cvat-data are currently JS written. Isn't it a problem?

Reply:
I remove this line from .nycrc. And add to package.json @istanbuljs/nyc-config-babel. 
As we discussed earlier, this has had a positive effect on file instrumentation. In particular, a larger number of .tsx files were included in the instrumentation.